dockerfiles/platform-build-centos-7/Dockerfile

```diff
@@ -62,15 +52,33 @@ ENV LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
 ENV PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
 ENV ACLOCAL_PATH=/usr/local/share/aclocal:$ACLOCAL_PATH
 
-RUN curl -L https://npmjs.org/install.sh | bash
+RUN echo "source /opt/rh/devtoolset-9/enable" >> /etc/bashrc
+SHELL ["/bin/bash", "--login", "-c"]
 
-ENV VCPKG_BINARY_SOURCES="clear;nuget,GitHub,read"
-ENV VCPKG_NUGET_REPOSITORY=https://github.com/hpcc-systems/vcpkg
+RUN . <(curl https://aka.ms/vcpkg-init.sh -L)
+RUN . ~/.vcpkg/vcpkg-init fetch cmake
+RUN ln -s `. ~/.vcpkg/vcpkg-init fetch cmake | tail -n 1` /usr/local/bin/
+RUN ln -s `. ~/.vcpkg/vcpkg-init fetch node | tail -n 1` /usr/local/bin/
+RUN ln -s `. ~/.vcpkg/vcpkg-init fetch node | tail -n 1 | sed -e 's!bin/node!bin/npm!g'` /usr/local/bin/
+
+# Libraries  ---
+RUN yum install -y \
+    curl-devel \
+    python3-devel \
+    ncurses-devel \
+    libmemcached-devel \
+    openldap-devel \
+    numactl-devel \
+    heimdal-devel \
+    java-11-openjdk-devel \
+    sqlite-devel \
+    libevent-devel \
+    v8-devel
 
 WORKDIR /hpcc-dev
 
 ARG GITHUB_ACTOR=hpcc-systems
-ARG BUILD_TAG=none
+ARG BUILD_TAG=candidate-8.8.0
```

Comment:
question: is this default needed?

Reply:
No - but having a working default is useful while developing.